.travis.yml

```diff
@@ -16,14 +16,41 @@ jobs:
   - python: 3.9-dev
   - <<: *latest_py3
     env: TOXENV=docs
+    #PPC64LE
+  - arch: ppc64le
+    python: pypy3
+  - arch: ppc64le
+    python: 3.5
+  - &latest_py3_ppc
+    arch: ppc64le
+    python: 3.8
+  - <<: *latest_py3_ppc
+    env: LANG=C
+  - arch: ppc64le
+    python: 3.9-dev
+  - <<: *latest_py3_ppc
+    env: TOXENV=docs
   allow_failures:
   # suppress failures due to pypa/setuptools#2000
   - python: pypy3
+  - arch: ppc64le
+    python: pypy3
   - <<: *latest_py3
     env: TOXENV=docs
+  - <<: *latest_py3_ppc
+    env: TOXENV=docs
+
```

Comment:
I think this should be reverted together with https://github.com/pypa/setuptools/pull/2379/files#r491708329

Reply:
Done.